lib/msal-common/src/utils/Constants.ts

```diff
@@ -216,4 +216,40 @@ export enum GrantType {
     DEVICE_CODE_GRANT = "device_code"
 };
 
+/**
```

Comment:
Can we add a link to the docs for the cache schema in the comments somewhere?

Reply:
Yes. Once they are ready. Can we delink this from this PR? As we synced up offline, docs effort is already kickstarted.